### PR TITLE
Enable column comment

### PR DESCRIPTION
**Pull request by satom9to5**
Motivation:
I want to print column comments on `db/schema.rb` files.
It is default feature of rails5 since rails/rails#22911

### CODE REVIEW

File: lib/active_record/connection_adapters/postgis/spatial_column.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def initialize(name, default, sql_type_metadata = nil, null = true, table_name =
             # @geometric_type = geo_type_from_sql_type(sql_type)
             build_from_sql_type(sql_type_metadata.sql_type)
           end
-          super(name, default, sql_type_metadata, null, table_name, default_function, collation)
+          super(name, default, sql_type_metadata, null, table_name, default_function, collation, comment: comment.presence)
```
Comment (teeparham) [inline lib/active_record/connection_adapters/postgis/spatial_column.rb]: I don't think we need `.presence`.

Comment (satom9to5) [reply lib/active_record/connection_adapters/postgis/spatial_column.rb]: Surely. fix it.

**Comment by satom9to5** [comment]
Sorry...I missed. I added test on `test/ddl_test.rb`.

**Comment by hirokishirai** [comment]
@satom9to5 Thanks for your nice change! this is super help for me :)
@teeparham I think you can merge this PR:)

**Comment by teeparham** [comment]
Released in 5.2.0, 4.1.1

**Pull request merged**